### DISCUSSION TITLE
XData are not read

### DISCUSSION THREAD

**Discussion by nickger** [haplokuon/netDxf#368] [Q&A] [answered]
To correctly identify dynamic blocks inserts, one need to read the value of "AcDbBlockRepBTag" XData. Unfortunately, it seems there are no XData read from dxf. Or I miss something?
[Drawing1.txt](https://github.com/haplokuon/netDxf/files/8215827/Drawing1.txt)

**Comment by haplokuon** [comment, marked as answer]
You will find it under the Block.Record extended data. Read [this](https://github.com/haplokuon/netDxf/discussions/340).

**Reply by nickger** [reply]
Exactly what I'm after, thx.
